File: app/models/scientist_profile.rb
class ScientistProfile < ActiveRecord::Base
  belongs_to :scientist
  #validates_presence_of :company, :department
  validates_presence_of :phone1, :email, :address1, :city, :state, :zip_code, :emphasis
  validates_uniqueness_of :scientist_id, :email

  xss_foliate :except => [:emphasis, :summary]
  xss_foliate :strip => [:emphasis, :summary]
end

File: app/assets/javascripts/ember-app/models/scientist_profile.js
var ScientistProfile = DS.Model.extend({

  prefix: DS.attr('string'),

  department: DS.attr('string'),

  department_url: DS.attr('string'),

  company: DS.attr('string'),

  address1: DS.attr('string'),

  address2: DS.attr('string'),

  city: DS.attr('string'),

  state: DS.attr('string'),

  zip_code: DS.attr('number'),

  phone1: DS.attr('string'),

  phone2: DS.attr('string'),

  email: DS.attr('string'),

  summary: DS.attr('string'),

  emphasis: DS.attr('string'),
  scientist: DS.belongsTo('Scientist')


});

module.exports = ScientistProfile;

